src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
I actually found liftW syntax useful, as you can do both liftW and liftO. 
In fact I would like to have whole bunch of `O` and `W` combinators added, they work pretty well if you work with writer. I can put you some source of them in comment, so you can perhaps add them, if you are interested?